platform/android/export/export_plugin.cpp

```diff
@@ -1706,9 +1706,20 @@ void EditorExportPlatformAndroid::_copy_icons_to_gradle_project(const Ref<Editor
 		const Ref<Image> &p_main_image,
 		const Ref<Image> &p_foreground,
 		const Ref<Image> &p_background,
-		const Ref<Image> &p_monochrome) {
+		const Ref<Image> &p_monochrome,
+		const String &icon_xml_path) {
 	String gradle_build_dir = ExportTemplateManager::get_android_build_directory(p_preset);
 
+	// This template string must always match the content of icon.xml file.
+	const String ICON_XML_TEMPLATE = 
```

Comment:
This constant can be moved at the top level alongside https://github.com/godotengine/godot/blob/d741a646a5ee20e77c1d0d9d4fbfed03d7b8dcc3/platform/android/export/export_plugin.cpp#L222